### ISSUE TITLE
Allow to change instances on the fly instead of on/off

### ISSUE THREAD

**Issue by Ileca** [SimonBrazell/privacy-redirect#215]
I come from Invidition that recommended your extension after stopping development and one thing that stopped me from using Privacy Redirect is the lack of on-the-fly instance switching. By that I mean the menu from More Options directly on the button. The thing is, invidious' instances are _very very_ unstable, meaning I have to change instances on a daily basis to find a workable one! With Privacy Redirect, I have to keep the option's menu as a tab to do that, which is really cumbersome. On the other hand, when you have decided which website to redirect, I don't see people playing with the on/off switches. Ok, maybe I wrong on this one, but you can still put the lists under each related line on the button, or at least add that for invidious alone as the other services are pretty stable afaik. Thanks!

**Comment by SimonBrazell** [comment]
Hi @Ileca the instances used to be present in the popup but it was suggested that I remove them as more services were added and it began to be cluttered. Perhaps I could add a collapsible menu that included the invidious instance to the popup?

**Comment by Ileca** [comment]
Yes, an instance menu for the invidious instance alone would be greatly appreciated.

I don't know how the popup was before you removed the instances from it, but if it was basically what you can see in the option's General tab, with buttons on top of the list of instances, I can understand that. However, [this part alone](https://i.imgur.com/PbM3qF2.png) looks good. You could put little checkboxes the size of the lines, like I awkwardly did. Another solution could be to make an option to allow the user to choose what service he wants on the button. I only use three of them so I, as an example, could gain some space. You could combine that with an option to select which instance menus they want to appear too.

**Comment by uuksu** [comment]
This same thing seems to happen with some Bibliogram instances that they just stop working and I have to find a new working one. The way the extension is now implemented forces me to copy paste the original url to browser and patiently to this random amount of times to get directed to working instance. It would be cool if I could choose one instance that I know currently works instead of trying to win working instance in the instance roulette 😀.
